modules/contact-form.php

```diff
@@ -14,3 +14,7 @@
  */
 
 include dirname( __FILE__ ) . '/contact-form/grunion-contact-form.php';
+
+if ( is_admin() ) {
```

Comment:
What about a filter here to disable the new feature. I would specifically note in the inline docs (and maybe in the filter name) that it is a temporary filter. Maybe say we'll keep it in for the next three? six? months _or_ unless there is a security issue in the older code that we opt to remove it before that time, etc etc etc.